.github/workflows/community-plugin-cd.yml

```diff
@@ -0,0 +1,47 @@
+on:
+  workflow_call:
+    inputs:
+      golang-version:
+        default: "1.21"
+        description: |
+          Set the version for golang
+        required: false
+        type: string
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: cd/checkout-repo
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+        with:
+          fetch-depth: 0
+
+      - name: cd/setup-plugin-environment
+        uses: mattermost/actions/plugin-ci/setup@main
+        with:
+          golang-version: ${{ inputs.golang-version }}
+
+      - name: cd/build-plugin
+        uses: mattermost/actions/plugin-ci/build@main
+
+  release-github:
+    runs-on: ubuntu-latest
+    needs:
+      - build
+    steps:
+      - name: cd/checkout-repository
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+        with:
+          fetch-depth: "0"
+
+      - name: cd/download-artifact
+        uses: actions/download-artifact@c850b930e6ba138125429b7e5c93fc707a7f8427 # v4.1.4
+        with:
+          name: dist
+          path: dist
+
+      - name: ci/publish-release
+        working-directory: dist
+        run: |
+          gh release create "$GITHUB_REF_NAME" --title "$GITHUB_REF_NAME" --notes-file release-notes.md  *.tar.gz
```

Comment:
I did test this job using the webex plugin and it failed with (See https://github.com/mattermost/mattermost-plugin-webex/actions/runs/8660349876/job/23748214542)

```
gh: To use GitHub CLI in a GitHub Actions workflow, set the GH_TOKEN environment variable. Example:
  env:
    GH_TOKEN: ${{ github.token }}
```

Could you please look into that?